datafusion/sqllogictest/test_files/window.slt

```diff
@@ -1135,8 +1135,8 @@ SELECT
 query IRR
 SELECT
   c8,
-  CUME_DIST() OVER(ORDER BY c9) as cd1,
-  CUME_DIST() OVER(ORDER BY c9 ROWS BETWEEN 10 PRECEDING and 1 FOLLOWING) as cd2
+  cume_dist() OVER(ORDER BY c9) as cd1,
+  cume_dist() OVER(ORDER BY c9 ROWS BETWEEN 10 PRECEDING and 1 FOLLOWING) as cd2
```

Comment:
necessary?

Reply:
It is consistent with conversion of other user-defined window functions.